lib/file.go

```diff
@@ -58,51 +58,77 @@ func pathKind(path string) string {
 	return "file"
 }
 
-// ReadDatasetFile decodes a dataset document into a Dataset
-func ReadDatasetFile(path string) (ds *dataset.Dataset, err error) {
-	var (
-		resp *http.Response
-		f    *os.File
-		data []byte
-	)
+// ReadDatasetFiles decodes a dataset document into a Dataset
```

Comment:
Outdated comment. It'd be great if this function comment broke down some of the nuances of reading multiple filepaths into a dataset, keeping it as high-level as possible. Something like this (adjusting for accuracy):

```go
// ReadDatasetFiles combines resources at zero or more filepaths, deserializing them into a
// *dataset.Dataset. Files can separately specify different components of a dataset
// but two provided components cannot overlap (modify the same component)
```

Reply:
Done.